information-security/midterm-2-study-guide.md

```diff
@@ -945,12 +945,12 @@ Page 256
 
 **True or False?**
 
-Maintaining Access is backdoor that is hard to detect because it modifies machine level code.
+One of many intruder behaviors is Maintaining Access is by adding a machine code backdoor that is hard to detect because it modifies machine level code.
```

Comment:
Can you clean up the grammar/capitalization for this sentence?

"One of many intruder behaviors is Maintaining Access is by adding a machine code backdoor that is hard to detect because it modifies machine level code."

Reply:
Allright, try that.  Yeah, my proposal was a bit rough.  Can change further if so desired.

Reply:
Also feel free to tweak if it's almost there by not quite.